Makefile

```diff
@@ -25,15 +25,16 @@ PWD=$(shell pwd)
 # All source code and documents, used when checking for misspellings
 ALLDOC := $(shell find . \( -name "*.md" -o -name "*.yaml" \) -type f | sort)
 
+TOOLS_MOD_DIR := ./internal/tools
```

Comment:
Using the tool package resolves issues with the setup-environment cache key not being updated on tool version changes.